wiki/Help_centre/Silence/en.md

```diff
@@ -30,15 +30,25 @@ After a user is silenced, an alert will be displayed in place of the chat input
 
 If you believe you were silenced unfairly, contact the [account support team](/wiki/People/Account_support_team#accounts@ppy.sh) at [accounts@ppy.sh](mailto:accounts@ppy.sh) to contest the silence. Make sure to do it from the email linked to your osu! account; mention your osu! username, as well as the approximate date and time when the silence was handed out.
 
-## Common reasons for silences
+## Common reasons for silences and durations
 
 *Caution: Engaging in "inappropriate behaviour" in private messages is not against the rules, unless said inappropriate behaviour is being delivered to unwilling or unconsenting recipients. If reported, appropriate punishments will be placed.*
 
-- Inappropriate behaviour in public chats or forums (e.g. `#multiplayer`, beatmap discussions, comments, etc.), including — but not limited to — spamming, nonsense chat, racism, etc.
-- Uploading or posting inappropriate content via other means (e.g. profile avatars, beatmap submissions, profile details, user page bio, banner, etc.).
+| Silence reason | Duration |
+| :-- | :-- |
+| Spam or inappropriate discussions in chat | 5 minutes |
+| Inappropriate comments, beatmap discussion posts, or forum posts | 80 minutes |
+| Racism in chat | 80 minutes |
+| Inappropriate links or visual content | 43 hours[^exact-duration] |
+| Severe misconduct | 43 hours[^exact-duration] |
```

Comment:
```suggestion
```

this one is so vague that I think it should be removed from the table, it doesn't help me understand silences any more than the sentence that's already below this